src/main/java/mate/academy/internetshop/dao/ItemDao.java

```diff
@@ -0,0 +1,20 @@
+package mate.academy.internetshop.dao;
+
+import mate.academy.internetshop.model.Item;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface ItemDao {
+    Item create(Item item);
+
+    Optional<Item> get(Long itemId);
+
+    Item update(Item item);
+
+    boolean delete(Long itemId);
```

Comment:
```suggestion
    boolean deleteById(Long itemId);
```
the same in each interface